docs/guide/cursors.txt

```diff
@@ -1,3 +1,164 @@
 =======
 Cursors
 =======
+
+When a driver executes a read that returns mutliple documents, the server does not immediately return all values that match the query. Instead, the driver creates a **Cursor** object which fetches the documents in batches.
+
+Where are cursors used in the driver?
+-------------------------------------
+
+The following functions directly return cursors to the user:
+
+
+* ``Collection.prototype.find``
+* ``Collection.prototype.aggregate``
+* ``Collection.prototype.listIndexes``
+* ``Db.prototype.aggregate``
+* ``Db.prototype.listCollections``
+
+In addition, many other methods, like ``Collection.prototype.findOne`` and ``Collection.prototype.watch``\ , use cursors in order to return results.
+
+How do I use a cursor?
+----------------------
+
+There are multiple ways to consume a cursor:
+
+Stream API
+^^^^^^^^^^
+
+All cursors in the Node Driver are Node Readable Streams operating in Object Mode. Cursors will work with most Node stream APIs.
+
+.. code-block:: js
+
+   // Using the stream API
+   const cursor = collection.find({});
+   cursor.pipe(new stream.Writable({
+     write: function(doc, _, callback) {
```

Comment:
can you convert this to es6 shorthand function:

from   `write: function(doc, _, callback) {`
to `write(doc, _, callback) {`